ethereum/weaksubjectivity/src/main/java/tech/pegasys/teku/weaksubjectivity/WeakSubjectivityCalculator.java

```diff
@@ -29,22 +34,25 @@
  * https://github.com/ethereum/eth2.0-specs/blob/weak-subjectivity-guide/specs/phase0/weak-subjectivity.md
  */
 public class WeakSubjectivityCalculator {
-  private static final UInt64 WITHDRAWAL_DELAY =
-      UInt64.valueOf(Constants.MIN_VALIDATOR_WITHDRAWABILITY_DELAY);
+  private static final UInt64 ETH_TO_GWEI = UInt64.valueOf(BigInteger.TEN.pow(9));
```

Comment:
seems like this should be a constant in the spec area?

Reply:
Sure - moved this to a new `EthConstants` class.  Conversion rates should never change, so these can just be static constants.